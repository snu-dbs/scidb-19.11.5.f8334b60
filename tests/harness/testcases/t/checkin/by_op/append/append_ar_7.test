# Append array with overlap at source, dest, or both.
--setup

store(build(<v:int64>[i=0:19:0:4], i), A)
store(build(<v:int64>[i=0:19:2:4], i), AA)

create array X<v:int64>[j]
create array XX<v:int64>[j=0:*:4:*]

--test
--echo Initial sums...
op_sum(A, v)
op_sum(AA, v)

--start-query-logging

append(A, X)
op_sum(X, v)
append(A, X)
op_sum(X, v)

append(AA, X)
op_sum(X, v)
append(AA, X)
op_sum(X, v)

append(A, XX)
op_sum(XX, v)
append(A, XX)
op_sum(XX, v)

append(AA, XX)
op_sum(XX, v)
append(AA, XX)
op_sum(XX, v)

--stop-query-logging

--cleanup
remove(A)
remove(AA)
remove(X)
remove(XX)
