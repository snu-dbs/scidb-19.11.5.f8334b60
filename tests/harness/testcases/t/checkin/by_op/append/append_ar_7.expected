Query was executed successfully

Query was executed successfully

Query was executed successfully

Query was executed successfully

Initial sums...
{i} v_sum
{0} 190

{i} v_sum
{0} 190

SCIDB QUERY : <append(A, X)>
Query was executed successfully

SCIDB QUERY : <op_sum(X, v)>
{i} v_sum
{0} 190

SCIDB QUERY : <append(A, X)>
Query was executed successfully

SCIDB QUERY : <op_sum(X, v)>
{i} v_sum
{0} 380

SCIDB QUERY : <append(AA, X)>
Query was executed successfully

SCIDB QUERY : <op_sum(X, v)>
{i} v_sum
{0} 570

SCIDB QUERY : <append(AA, X)>
Query was executed successfully

SCIDB QUERY : <op_sum(X, v)>
{i} v_sum
{0} 760

SCIDB QUERY : <append(A, XX)>
Query was executed successfully

SCIDB QUERY : <op_sum(XX, v)>
{i} v_sum
{0} 190

SCIDB QUERY : <append(A, XX)>
Query was executed successfully

SCIDB QUERY : <op_sum(XX, v)>
{i} v_sum
{0} 380

SCIDB QUERY : <append(AA, XX)>
Query was executed successfully

SCIDB QUERY : <op_sum(XX, v)>
{i} v_sum
{0} 570

SCIDB QUERY : <append(AA, XX)>
Query was executed successfully

SCIDB QUERY : <op_sum(XX, v)>
{i} v_sum
{0} 760

Query was executed successfully

Query was executed successfully

Query was executed successfully

Query was executed successfully

